src/wms/src/main/java/org/geoserver/wms/map/png/PNGJWriter.java

```diff
@@ -51,7 +51,7 @@ public RenderedImage writePNG(
         try {
             output = writer.writePNG(image, outStream, quality, filterType);
         } catch (Exception e) {
-            LOGGER.log(Level.SEVERE, "Failed to encode the PNG", e);
+            LOGGER.log(Level.FINER, "Failed to encode the PNG", e);
```

Comment:
No... it should check if there is a ClientStreamAbortedException wrapped, or something else. Check how the dispatcher does it.
https://github.com/geoserver/geoserver/blob/5afddea76c9b95c2a9005ebb3957338719f488c3/src/ows/src/main/java/org/geoserver/ows/Dispatcher.java#L1679

Reply:
A simple approach could be to just avoid logging and have the exception go up the chain, it will get logged by the Dispatcher if it's not a client stream aborted exception, e.g.:

```
       } catch (Exception e) {
            throw new ServiceException("Failed to encode the PNG", e);
        }
```